orchagent/portsorch.cpp

```diff
@@ -7986,6 +7986,8 @@ void PortsOrch::updatePortOperStatus(Port &port, sai_port_oper_status_t status)
                     isUp ? "up" : "down");
         }
     }
+    SWSS_LOG_NOTICE("Updating the nexthop for port %s and operational status %s", port.m_alias.c_str(), isUp ? "up" : "down");
```

Comment:
Can you change this to INFO? There is no useful info here. Also can limit logs during interface flaps

Reply:
updated in the latest commit.